docarray/typing/tensor/tensor.py

```diff
@@ -22,26 +22,66 @@ def __get_validators__(cls):
 
     @classmethod
     def validate(
-        cls: Type[T], value: Union[T, Any], field: 'ModelField', config: 'BaseConfig'
+        cls: Type[T],
+        value: Union[T, np.ndarray, List[Any], Tuple[Any], Any],
+        field: 'ModelField',
+        config: 'BaseConfig',
     ) -> T:
         if isinstance(value, np.ndarray):
             return cls.from_ndarray(value)
         elif isinstance(value, Tensor):
             return cast(T, value)
+        elif isinstance(value, list) or isinstance(value, tuple):
+            try:
+                arr_from_list: np.ndarray = np.asarray(value)
+                return cls.from_ndarray(arr_from_list)
+            except Exception:
+                pass  # handled below
         else:
             try:
                 arr: np.ndarray = np.ndarray(value)
                 return cls.from_ndarray(arr)
             except Exception:
                 pass  # handled below
-        raise ValueError(f'Expected a numpy.ndarray, got {type(value)}')
+        raise ValueError(f'Expected a numpy.ndarray compatible type, got {type(value)}')
 
     @classmethod
     def from_ndarray(cls: Type[T], value: np.ndarray) -> T:
         return value.view(cls)
 
+    @classmethod
+    def __modify_schema__(cls, field_schema: Dict[str, Any]) -> None:
+        # this is needed to dump to json
+        field_schema.update(type='string', format='tensor')
+
+    def _to_json_compatible(self) -> np.ndarray:
```

Comment:
Same for this; needed in the abstract class?

Reply:
This one yes we might ask for it even if in some case it will just return self